packages/modal/src/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const footerStyle = css`
+  position: relative;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+
+  display: flex;
+  justify-content: right;
+  flex-direction: row-reverse;
+  padding: 16px 24px;
+  border-top: 1px solid ${uiColors.gray.light2};
+
+  & > * {
```

Comment:
We should rework this selector to not use `*`. This selector will look at every element on the page to check to see if the parent matches the selector, and it's generally bad practice to do this outside of narrow use-cases. Here, I don't think there's justification for using it.

Reply:
Yeah on second thought it also makes sense to leave the styling of the footer contents to the consumer. I pushed these styles up to the modal.